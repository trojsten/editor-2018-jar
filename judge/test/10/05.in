INT:
velkostmapy
9
INT:
dlzka
0
INT:
vyska
0
INT:
okuliare
0
VECTOR_STR:
mapa
9
*08/M@GL$
#####LVB/
#K%U#XOXI
#?SH#@U3$
#####13+L
OH%%<M,VG
--,8FS4E>
10-O$;PD5
D@6&M5>>[
