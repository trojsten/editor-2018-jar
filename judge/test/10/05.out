{"dlzka": 5, "vyska": 4}